### PR TITLE
Ansible Job Update- Vmware

### PR DESCRIPTION
**Pull request by naveena-maplelabs**
Issue:
During vmware job update, include vms overwrite the existing vms
available in the job.

Fix:
Flag is added(replace_existing), if flag is set to true, existing job vms
will be replaced with new include vms.
If flag is set to false, new include vms will be added to existing
vms aavilable in the job.

### CODE REVIEW

File: library/cohesity_job.py [hunk 1]
```diff
@@ -741,6 +741,14 @@ def update_vmware_job(module, job_meta_data, job_details):
         if len(module.params.get('include')) != 0:
             vms = module.params.get('include')
             include_vm_ids = get_vmware_ids(module, job_meta_data, job_details, vms)
+            remove_existing = module.params.get('remove_existing')
```
Comment (pyashish) [inline library/cohesity_job.py]: Instead of remove_existing ,can we do replace_existing ?

File: tasks/job.yml [hunk 2]
```diff
@@ -14,6 +14,7 @@
     storage_domain: "{{ cohesity_protection.storage_domain | default('DefaultStorageDomain') }}"
     delete_backups: "{{ cohesity_protection.delete_backups | default(False) }}"
     cancel_active: "{{ cohesity_protection.cancel_active | default(False) }}"
+    remove_existing: "{{ cohesity_protection.remove_existing | default(True) }}"
```
Comment (pyashish) [inline tasks/job.yml]: what is current behavior ? lets default to that

File: library/cohesity_job.py (hunk 1 above)
Comment (naveena-maplelabs) [reply library/cohesity_job.py]: Okay

File: tasks/job.yml (hunk 2 above)
Comment (naveena-maplelabs) [reply tasks/job.yml]: Currently, updating VMware job will replace available VMs in the job. So flag is to set to true, will replace all existing vms.

**Pull request merged**